plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -831,6 +831,7 @@ public Answer cloneVolumeFromBaseTemplate(CopyCommand cmd) {
 
             DatastoreMO dsMo = new DatastoreMO(context, morDatastore);
 
+            String cloneVMName = volume.getVmName();
```

Comment:
@nvazquez cc @DaanHoogland - in this method we can actually clone the VM as the instance name is passed to the volume and can be obtained via `volume.getVmName()`... something we can explore/use. If during template register + OVF parsing we know the disks then we can create DB records for all the volumes of the VM (like ROOT-xyz, DATA-xyz-1, DATA-xyz2...) before the StartCommand...